scalableminds/webknossos
# Make isValidNewName route always return 200

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- click on "Add Dataset" in dashboard
- type a name into the "Dataset Name" field that already exists for another ds -> red label should appear
- change name to something unique -> error should disappear
- no error toast should appear during the above

### TODOs:
- [x] Since it is not user facing, no Changelog, only migration?

### Issues:
- fixes #7399

------
(Please delete unneeded items, merge only when none are left open)
